packages/react/service_worker/OidcServiceWorker.ts

```diff
@@ -321,8 +322,12 @@ const handleMessage = (event: ExtendableMessageEvent) => {
     case 'init': {
       const oidcServerConfiguration = data.data.oidcServerConfiguration;
       let trustedDomain = trustedDomains[configurationName];
-      
-      const domains = Array.isArray(trustedDomain) ? trustedDomain : trustedDomain.domains;
+      let domains: Domain[];
```

Comment:
Consider extracting the logic to a function

Reply:
Extracted to one function handling both cases